### DISCUSSION TITLE
How to connect to any wallet from the list in our dApp (React Native App)?

### DISCUSSION THREAD

**Discussion by aydil45** [WalletConnect/walletconnect-monorepo#1949] [Developer Support]
I want to create a decentralized app in react native that can allow the user to connect to any wallet displayed in the list of wallets. I want to accomplish this goal but I have no clear idea about how to do it. Please enlighten me on whether it is possible or not in react native and if it is possible then how to do it.

**Comment by crypblizz8** [comment]
Hi there, this approach is possible, feel free to check out our examples here: https://github.com/WalletConnect/react-native-examples

I recommend checking the dapp folder. Feel free to reach out on [Discord](https://discord.gg/kdTQHQ6AFQ) with any questions or here
